blockchain/standalone/treasury_test.go

```diff
@@ -4,25 +4,28 @@
 
 package standalone
 
-import "testing"
+import (
+	"errors"
+	"testing"
+)
 
 func TestTSpendExpiryNegative(t *testing.T) {
 	// 5 is not a valid start for a tvi of 11 with a mul of 3.
 	_, err := CalculateTSpendWindowStart(5, 11, 3)
 	if e, ok := err.(RuleError); !ok {
 		t.Fatal(err)
-	} else if e.ErrorCode != ErrTSpendStartInvalidExpiry {
+	} else if !errors.Is(e, ErrTSpendStartInvalidExpiry) {
```

Comment:
Same here.  No need for the extra checks.  Just `if !errors.Is(err, ErrTSpendStartInvalidExpiry) {`.